docs/data-cookbooks/browse-generation.md

```diff
@@ -88,9 +88,11 @@ For this example, you are going to be adding two workflows to your Cumulus deplo
 
   The output from this step will be passed into the 'ProcessingStep', which in this example will utilize a 'fake' processing lambda we provide for testing/as an example in Core, however to use your own data you will need to write a lambda that generates the appropriate CMR metadata file and accepts and returns appropriate task inputs and outputs.
 
-  From that step we will utilize the core task ```MoveGranules``` to move the granules to the target buckets as defined in the collection configuration.  That step will transfer the files to their final storage location and update the payload.
+  From that step we will utilize a core task ```FilesToGranulesStep``` that will transform the processing output event.input list/config.InputGranules into an array of Cumulus [granules](https://github.com/nasa/cumulus/blob/master/packages/api/models/schemas.js) objects.
 
-  That payload will be used in the ```CmrStep``` combined with the previously generated CMR file to export the granule metadata to CMR.
+  Using the generated granules list, we will utilize the core task ```MoveGranules``` to move the granules to the target buckets as defined in the collection configuration.  That step will transfer the files to their final storage location and update the CMR metadata files and the granules list as output.
```

Comment:
`MoveGranules` || `MoveGranulesStep`? It's also `MoveGranuleStep` below, even though the step _can_ move multiple granules.

Reply:
The plural form is probably a larger nitpick than just this doc though, you can ignore that.

Reply:
I think this is fixed in the above change 72b733b.